ingestor/src/main/resources/reference.conf

```diff
@@ -16,5 +16,5 @@ data-target.cluster = {
    resolveSecrets = false
 }
 data-target.index.name=authoritative-concepts
-data-target.bulk.max-records=50000
+data-target.bulk.max-records=10000
```

Comment:
Reduce the max number of records in each request to 10,000 because setting it to 50,000 overwhelms the ES cluster.